pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -1708,6 +1709,154 @@ func GetAvailableNotifiers() []*NotifierPlugin {
+			Name:        "Jira",
+			Description: "Creates Jira issues from alerts",
+			Heading:     "Jira settings",
+			Options: []NotifierOption{
+				{
+					Label:        "API URL of Jira instance, including version of API",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "https://grafana.atlassian.net/rest/api/3",
+					PropertyName: "api_url",
+					Description:  "Supported v2 or v3 APIs, e.g. https://grafana.atlassian.net/rest/api/3",
+					Required:     true,
+				},
+				{
+					Label:        "HTTP Basic Authentication - Username",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					PropertyName: "user",
+					Description:  "Username to use for Jira authentication. Usually, it should be your email address.",
+					Secure:       true,
+					Required:     false,
+				},
+				{
+					Label:        "HTTP Basic Authentication - Password",
+					Element:      ElementTypeInput,
+					InputType:    InputTypePassword,
+					PropertyName: "password",
+					// Go to https://id.atlassian.com/manage-profile/security/api-tokens to obtain a token.
+					Description: "Password to use for Jira authentication.",
+					Secure:      true,
+					Required:    false,
+				},
+				{
+					Label:        "Authorization Header - Personal Access Token",
+					Element:      ElementTypeInput,
+					InputType:    InputTypePassword,
+					PropertyName: "api_token",
+					// Go to https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html for how to obtain a token.
+					Description: "Personal Access Token that is used as a bearer authorization header.",
+					Secure:      true,
+					Required:    false,
+				},
+				{
+					Label:        "Project Key",
+					Description:  "The project key where issues are created",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "Grafana",
+					PropertyName: "project",
+					Required:     true,
+				},
+				{
+					Label:        "Issue Type",
+					Description:  "The type of the Jira issue (e.g., Bug, Task, Story). You can use templates to customize this field.",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "Task",
+					Required:     true,
+					PropertyName: "issue_type",
+				},
+				{
+					Label:        "Summary",
+					Description:  fmt.Sprintf("The summary of the Jira issue. You can use templates to customize this field. Maximum length is %d characters. Otheriwse, the text will be truncated", jira.MaxSummaryLenRunes),
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  jira.DefaultSummary,
+					PropertyName: "summary",
+				},
+				{
+					Label:        "Description",
+					Description:  fmt.Sprintf("The description of the Jira issue. You can use templates to customize this field. Maximum length is %d characters. Otheriwse, the text will be truncated", jira.MaxDescriptionLenRunes),
```

Comment:
```suggestion
					Description:  fmt.Sprintf("The description of the Jira issue. You can use templates to customize this field. Maximum length is %d characters.", jira.MaxDescriptionLenRunes),
```
same as above, the truncated text is implied by the maximum length conditions.